src/test/java/uk/gov/companieshouse/overseasentitiesapi/service/AddressUtils.java

```diff
@@ -0,0 +1,80 @@
+package uk.gov.companieshouse.overseasentitiesapi.service;
```

Comment:
Could this be moved into test? And renamed AddressTestUtils?

Reply:
Sure

Reply:
I'm assuming by this you mean put in its own utils package in test/java/ as it is already in the test package

The commit I'm about to push has done that though it can be changed if that isn't what you meant

Reply:
Yes, that's good. Just the class rename.